client/src/main/kotlin/no/nav/su/se/bakover/client/oppdrag/avstemming/AvstemmingRequest.kt

```diff
@@ -59,19 +60,22 @@ data class AvstemmingDataRequest(
         val avvistFortegn: Fortegn,
         val manglerAntall: Int,
         val manglerBelop: BigDecimal,
-        val manglerFortegn: Fortegn
+        val manglerFortegn: Fortegn,
     ) {
         fun totaltAntall() = godkjentAntall + varselAntall + avvistAntall + manglerAntall
     }
 
+    /**
+     * ID 140
+     */
     data class Detaljdata(
         val detaljType: Detaljtype,
         /*
             Hvem detaljen gjelder. Kan innehold dnr, fnr, TSS nummer eller orgnr. Tilsvarer oppdragGjelder
          */
         val offnr: String,
         val avleverendeTransaksjonNokkel: String,
-        val tidspunkt: String // TODO: Finne format på denne
+        val tidspunkt: String, // TODO: Finne format på denne
```

Comment:
Påminnelse. Antar dette var en av grunnen til at vi måtte test-deploye

Reply:
I utgangspunktet er dette urørt kode som kun er i sving for grensesnittsavstemming, tipper kommentaren bare er noe som henger igjen, da formatet for dette feltet er det samme som for andre "oppdrag timestamps".